rocketpy/Flight.py

```diff
@@ -1428,8 +1429,8 @@ def uDot(self, t, u, postProcessing=False):
                     M1 -= (compCp + a) * compLiftYB
                     M2 += (compCp + a) * compLiftXB
             # Calculates Roll Moment
-            if aerodynamicSurface["name"] == "Fins":
-                Clfdelta, Cldomega, cantAngleRad = aerodynamicSurface["roll parameters"]
+            if aerodynamicSurface.name == "Fins":
+                Clfdelta, Cldomega, cantAngleRad = aerodynamicSurface.rollParameters
```

Comment:
Should we apply EAFP here rather than LBYL?

https://realpython.com/python-lbyl-vs-eafp/

Reply:
Yes, I think we should, let me make a suggestion here, and you can say if it is clear enough or not: 

```suggestion
            try:
                Clfdelta, Cldomega, cantAngleRad = aerodynamicSurface.rollParameters
            except AttributeError:
                pass
```

However, I think we now should do something different, cheking the type of the aerodynamic surface, what do you think? And for that *we should import Fins class in the beggining of the file*

```suggestion
            if isinstance(aerodynamicSurface, Fins):
                Clfdelta, Cldomega, cantAngleRad = aerodynamicSurface.rollParameters
````